content/docs/start/data-pipelines.md

```diff
@@ -304,23 +306,23 @@ stages:
 DVC pipelines (`dvc.yaml` file, `dvc run`, and `dvc repro` commands) solve a few
 important problems:
 
-- _Automation_ - run a sequence of steps in a "smart" way that makes iterating
+- _Automation_: run a sequence of steps in a "smart" way which makes iterating
   on your project faster. DVC automatically determines which parts of a project
-  need to be run, and it caches "runs" and their results, to avoid unnecessary
-  re-runs.
-- _Reproducibility_ - `dvc.yaml` and `dvc.lock` files describe what data to use
+  need to be run, and it caches "runs" and their results (in the aptly-named
+  <abbr>run-cache</abbr>) to avoid unnecessary reruns.
+- _Reproducibility_: `dvc.yaml` and `dvc.lock` files describe what data to use
   and which commands will generate the pipeline results (such as an ML model).
   Storing these files in Git makes it easy to version and share.
-- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - describing
-  projects in way that it can be reproduced (built) is the first necessary step
-  before introducing CI/CD systems. See our sister project,
+- _Continuous Delivery and Continuous Integration (CI/CD) for ML_: describing
+  projects in way which it can be reproduced (built) is the first necessary step
```

Comment:
Should be "in a way that can be" I think. It had some typo before but it's a very restrictive clause. I just don't see how "which" works here even when I really want to let this go. It would require that "describing projects in a way" was a complete thought.

Reply:
didn't read this carefully. Should definitely remove "it" and make it simply "that."